### PR TITLE
[Draft] gui_utils Minor bugfix on change to...

### PR DESCRIPTION
**Pull request by Syres916**
...Draft Wb with grid enabled. Please backport to 0.19 branch, see discussion https://forum.freecadweb.org/viewtopic.php?f=3&t=60251

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by JOJ0** [comment]
Hi @Syres916, didn't know how to ping you on FreeCAD board. I have some more hints on broken imports. You might wann add them to this PR? https://forum.freecadweb.org/viewtopic.php?f=3&t=60251&p=517749
HTH

File: src/Mod/Draft/draftutils/gui_utils.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ def get_3d_view():
         Return `None` if the graphical interface is not available.
     """
     if App.GuiUp:
+        import FreeCADGui as Gui
+        from pivy import coin
```
Comment (marioalexis84) [inline src/Mod/Draft/draftutils/gui_utils.py:68]: @Syres916 those modules are already imported in this lines
https://github.com/FreeCAD/FreeCAD/blob/a049b9118be81246b754c8cd963e434ce8a6576b/src/Mod/Draft/draftutils/gui_utils.py#L48-L51

**Comment by Syres916** [comment]
@marioalexis84, if you read the forum bug report both myself and the OP have proved in certain circumstances those original imports are ignored hence me adding them into the specific function. It's been very difficult to replicate the bug but it is there.

**Comment by marioalexis84** [comment]
In that case the problem is more serious.
I tested the tag 0.19.2 and I did not find any problem.
Can you try replacing this in original lines where the modules are imported and load Draft workbench?
<pre>
App.Console.PrintWarning("App.GuiUp value: {}\n".format(App.GuiUp))
try:
    if App.GuiUp:
        import FreeCADGui as Gui
        from pivy import coin
        from PySide import QtGui
except Exception as ex:
    App.Console.PrintWarning(ex)
</pre>

It does not seem to be a problem with the import of modules, but just in case let's test it.
The value of App.GuiUp is set to 1 in FreeCADGuiInit.py

**Comment by Syres916** [comment]
There is no output to the Report View after the changes you suggested, therefore everything above `def get_3d_view():` is not triggered from what I can deduce.

**Comment by yorikvanhavre** [comment]
In any case this does not harm, there is no penalty in having a same import several times. And I think I recall having seen something similar already, that global imports were suddenly not available anymore inside functions... I propose to merge this as it fixes the issue, but let's keep an eye open to try to understand why this happens

**Comment by yorikvanhavre** [comment]
Thanks @Syres916 !! Merging

**Comment by bitfarmer** [comment]
Might be completely out of line here but I can reproduce this problem by setting the initially auto-loaded module to TechDraw whereas if I set it to Arch, the problem doesn't occur. If any one needs a backtrace please ask. I am on version OS: macOS 10.16
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.24291 (Git)
Build type: Release
Branch: (HEAD detached at 0.19.2)
Hash: 7b5e18a0759de778b74d3a5c17eba9cb815035ac
Python version: 3.8.8
Qt version: 5.12.9
Coin version: 4.0.0
OCC version: 7.4.0
Locale: C/Default (C)

**Pull request merged**